homeassistant/components/ozw/climate.py

```diff
@@ -331,20 +343,21 @@ def _get_current_mode_setpoint_values(self) -> Tuple:
 
     def _set_modes_and_presets(self):
         """Convert Z-Wave Thermostat modes into Home Assistant modes and presets."""
-        if not self.values.mode:
-            return
         all_modes = {}
         all_presets = {PRESET_NONE: None}
-        # Z-Wave uses one list for both modes and presets.
-        # Iterate over all Z-Wave ThermostatModes and extract the hvac modes and presets.
-        for val in self.values.mode.value[VALUE_LIST]:
-            if val[VALUE_ID] in MODES_LIST:
-                # treat value as hvac mode
-                hass_mode = ZW_HVAC_MODE_MAPPINGS.get(val[VALUE_ID])
-                all_modes[hass_mode] = val[VALUE_ID]
-            else:
-                # treat value as hvac preset
-                all_presets[val[VALUE_LABEL]] = val[VALUE_ID]
+        if not self.values.mode:
+            all_modes[HVAC_MODE_HEAT] = None
+        else:
+            # Z-Wave uses one list for both modes and presets.
+            # Iterate over all Z-Wave ThermostatModes and extract the hvac modes and presets.
+            for val in self.values.mode.value[VALUE_LIST]:
+                if val[VALUE_ID] in MODES_LIST:
+                    # treat value as hvac mode
+                    hass_mode = ZW_HVAC_MODE_MAPPINGS.get(val[VALUE_ID])
+                    all_modes[hass_mode] = val[VALUE_ID]
+                else:
+                    # treat value as hvac preset
+                    all_presets[val[VALUE_LABEL]] = val[VALUE_ID]
```

Comment:
Can we turn this around from `if not … else` to `if … else` ? 
```suggestion
        if self.values.mode:
            # Z-Wave uses one list for both modes and presets.
            # Iterate over all Z-Wave ThermostatModes and extract the hvac modes and presets.
            for val in self.values.mode.value[VALUE_LIST]:
                if val[VALUE_ID] in MODES_LIST:
                    # treat value as hvac mode
                    hass_mode = ZW_HVAC_MODE_MAPPINGS.get(val[VALUE_ID])
                    all_modes[hass_mode] = val[VALUE_ID]
                else:
                    # treat value as hvac preset
                    all_presets[val[VALUE_LABEL]] = val[VALUE_ID]
        else:
            all_modes[HVAC_MODE_HEAT] = None
```